src/NATS.Client.Core/Commands/PingCommand.cs

```diff
@@ -1,68 +1,8 @@
-using NATS.Client.Core.Internal;
-
 namespace NATS.Client.Core.Commands;
 
-internal sealed class PingCommand : CommandBase<PingCommand>
+internal sealed class PingCommand
```

Comment:
Is there value to making this a struct since it doesn't require inheritance?

Reply:
:+1: done